src/DynamoCore/Core/UndoRedoRecorder.cs

```diff
@@ -241,7 +242,16 @@ public XmlElement PopFromUndoGroup()
                     "UndoStack cannot be popped with non-empty RedoStack");
             }
 
-            return PopActionGroupFromUndoStack();
+            try
+            {
+                return PopActionGroupFromUndoStack();
+            }
+            catch (InvalidOperationException ex)
+            {
+                Log(ex);
```

Comment:
Is this logging to Dynamo Console?

Reply:
I think if the log(exception) is logged as error to the log file.